_targets.R

```diff
@@ -23,10 +23,10 @@ targets_list <- list(
   tar_target(grid_params, tibble(
     crs = "+proj=lcc +lat_0=45 + lon_0=-97 +lat_1=36 +lat_2=52 +x_0=0 +y_0=0 +ellps=WGS84 +units=m",
     cellsize = 25000,
-    xmin = -200000,
-    ymin = -1125000,
-    nx = 110,
-    ny = 85
+    xmin = -2700000,
+    ymin = -1750000,
+    nx = 217,
+    ny = 141
```

Comment:
Wow these are a big shift (25 grid cells in the y direction, and 100  in the x direction) -- are we no longer working solely with the midwest subset that the UW professor put together?

Reply:
Not sure - these were the new parameters associated with the grid for `iy` and `jx`. I am wondering if the new projection is off? Because plotting this results in this:

```r
library(targets)
library(sf)

tar_load(grid_cells_sf)
grid_bbox_sf <- st_as_sfc(st_bbox(grid_cells_sf))

midwest_sf <- st_as_sf(maps::map('state', plot=FALSE, fill=TRUE)) %>% 
  st_transform(st_crs(grid_cells_sf))

plot(st_geometry(grid_bbox_sf))
plot(st_geometry(midwest_sf), add=TRUE)
```

![image](https://user-images.githubusercontent.com/13220910/148813136-28735d6c-c89d-4c37-b3bf-8cd2d9fd432b.png)

Reply:
@jread-usgs bigger than the extent you shared in that Temp figure. That figure looks like what the original extent we were using.

Reply:
@wdwatkins pulling you in here to see if you know anything else about the grid differences between the original GCMs that were on GDP and the new debiased GCMs you added. I used the `iy` and `jx` variables to figure out the grid dimensions (see [here](https://github.com/USGS-R/lake-temperature-model-prep/issues/273#issuecomment-1007807437)) but the new debiased GCMs have a much much larger grid scale than the previous grid (see [this fig](https://github.com/USGS-R/lake-temperature-model-prep/pull/276#issuecomment-1009149738) Jordan shared below). 

Do you have any understanding of why these might be so different? Is my projection off for the new grid? And if so, how might I find out the CRS from the GDP grid?

Reply:
@hcorson-dosch and I tried to find the CRS from the new grid yesterday, but couldn't find one. Hayley explored the file using python and I tried with these R commands. Nothing is returned when it should be

```r
nc <- ncdf4::nc_open('http://gdp-netcdfdev.cr.usgs.gov:8080/thredds/dodsC/notaro_debias/files/debias_windspeed_mri_mid21_longerdaymet_ghcn_365d.nc')
ncdf4.helpers::nc.get.proj4.string(nc, "iy")
```

Reply:
Yeah, I don't see an explicit CRS anywhere in the debiased file I just checked.  You could sanity-check the extent by looking at the xlat/xlon variables though in the debiased files to take the projection out of the equation.  Just scrolling through the raw values your figure looks reasonable.

The GDP files have some metadata about grid origin etc in the global attributes.  You could also look at the raw lat/lon values there to confirm.
![image](https://user-images.githubusercontent.com/13987349/149390875-aab9043e-908d-49f4-b6d6-ceeb8cf9e9cb.png)


I would guess the GDP thredds data is subset to a particular study area if it is significantly smaller.

Reply:
Hmmm but that is the old data set (the not debiased one). The new dataset doesn't show any Global Attributes http://gdp-netcdfdev.cr.usgs.gov:8080/thredds/dodsC/notaro_debias_mri.html

Reply:
Right.  You can always compare the xlat/xlon value between the two datasets though, and compare those to what the maps of projected extent look like .  One or both of those datasets may be subset from some larger file, to the extents may not be the same, but it would not be unreasonable that they use the same coordinate system.